.github/workflows/ci.yaml

```diff
@@ -6,33 +6,32 @@ on:
   # Run daily at 0:01 UTC
   schedule:
     - cron: "1 0 * * *"
+  workflow_dispatch:
```

Comment:
Use this to run the workflows on demand if needed from https://github.com/ssl-hep/ServiceX_frontend/actions